network-api/networkapi/buyersguide/templates/buyersguide/catalog.html

```diff
@@ -67,9 +67,9 @@ <h1 class="h1-heading text-center mt-0">{% trans "Be Smart. Shop Safe." %}</h1>
     <div class="product-box-list d-flex justify-content-center align-items-stretch flex-wrap">
 
       {% for product in products %}
-        {% if product.is_current %}
+        {% if product.specific.is_current %}
```

Comment:
Not sure why (I haven't dug into it) but the products don't show up on the BuyersGuidePage when I run `migrate_products`. I suspect it has to do with this line. 

Edit: Confirmed, it's this line preventing products from showing up after the `migrate_products` script is run.

Reply:
I still see products, just not "all products" - it should only show 136 of them